docs/ecosystem-user-event.md

```diff
@@ -16,12 +16,12 @@ import React from 'react'
 import { render } from '@testing-library/react'
 import userEvent from '@testing-library/user-event'
 
-test('Fill text area', () => {
-  const { getByTestId } = render(<textarea data-testid="email" />)
-
-  userEvent.type(getByTestId('email'), 'Hello, World!')
-  expect(getByTestId('email')).toHaveAttribute('value', 'Hello, World!')
+const { getByRole } = test('click', () => {
+  render(<textarea />)
```

Comment:
Looks like the destructuring statement jumped to the line above:

```js
test('click', () => { 
  const { getByRole } = render(<textarea />) 
```

Reply:
Done